config.sh

```diff
@@ -723,6 +745,12 @@ if [ "$PLATFORM" = "psp" -o "$PLATFORM" = "gp2x" \
 	OVERLAY="false"
 fi
 
+if [ "$PLATFORM" = "emscripten" ]; then
+	echo "Force-enabling OpenGL ES support; force-disabling unsupported GL renderers (Emscripten)."
+	GLES="true"
+	GL_FIXED="false"
+fi
+
```

Comment:
This bit should probably instead be combined into the section at line 637 and prefaced with `Enabling Emscripten-specific hacks.` for better consistency with the other platforms.

Reply:
Done.